firebase-auth/src/iosMain/kotlin/dev/gitlive/firebase/auth/auth.kt

```diff
@@ -21,29 +19,21 @@ import platform.Foundation.*
 actual val Firebase.auth
     get() = FirebaseAuth(FIRAuth.auth())
 
-actual fun Firebase.auth(app: FirebaseApp) =
-    FirebaseAuth(FIRAuth.authWithApp(app.ios))
+@Suppress("CAST_NEVER_SUCCEEDS")
```

Comment:
eh?

Reply:
It works, its weird but Kotlin cocoapods imports transitive dependencies using a different name than they get when imported directly. However, the cast actually does succeed, since it is the same class